source/chapters/user_interface.rst

```diff
@@ -405,6 +404,9 @@ Transport Controls
  expletives on the fly, or simply as part of your mix.
 
 
+ .. note:: Not all of these controls may be present in all skins anymore (like the Deere skin). However, they can be mapped to keyboards and MIDI controllers. See this `guide here. <https://github.com/mixxxdj/mixxx/wiki/mixxxcontrols>`_
```

Comment:
```suggestion
 .. note:: Not all of these controls may be present in all skins anymore (like the Deere skin). However, they can still be mapped to keyboards and MIDI controllers.

Reply:
We'll add back the link as in internal reference in the 2.3 version of the manual.

Reply:
Okay.